packages/marshal/src/marshal-justin.js

```diff
@@ -445,3 +451,34 @@ const decodeToJustin = (encoding, shouldIndent = false, slots = []) => {
 };
 harden(decodeToJustin);
 export { decodeToJustin };
+
+/**
+ * @param {Passable} passable
+ * @param {boolean} [shouldIndent]
+ * @returns {string}
+ */
+export const passableAsJustin = (passable, shouldIndent = true) => {
+  let slotCount = 0;
+  // eslint-disable-next-line no-plusplus
+  const convertValToSlot = val => `s${slotCount++}`;
+  const { toCapData } = makeMarshal(convertValToSlot);
+  const { body, slots } = toCapData(passable);
+  const encoded = JSON.parse(body);
+  return decodeToJustin(encoded, shouldIndent, slots);
+};
+harden(passableAsJustin);
+
+/**
+ * qp for quote passable as a quasi-quoted Justin expression.
+ *
+ * Modelled on `quote` from `assert.js` in `'ses'`. But uses Justin
+ * instead of `bestEffortStringify`. The full name of this would have been
+ * `quotePassable`. But since the `quote` from `assert.js` is always used via
+ * its rename to `q`, we're skipping the rename and just
+ * naming this variable `qp`.
```

Comment:
When somebody looks up `qp`, I think it's less likely that they'll want to know this sort of information about how it's built and more likely that they'll want to know how to use it. An `@example` such as the one in the PR description would be nice.

Please be sure to note that it's not lazy.

I suggest moving these notes about how it's implemented to `//` comments.

Reply:
Much better, thanks!

Done